examples/demo-app/src/reducers/index.js

```diff
@@ -90,9 +90,16 @@ export const loadRemoteResourceSuccess = (state, action) => {
   // TODO: replace generate with a different function
   const datasetId = action.options.id || generateHashId(6);
   const {dataUrl} = action.options;
+
+  let data = action.response;
   let processorMethod = processRowObject;
+
   // TODO: create helper to determine file ext eligibility
-  if (dataUrl.includes('.json') || dataUrl.includes('.geojson')) {
+  if (dataUrl.includes('.csv')) {
```

Comment:
Could we look at the `shape` field of the returned data instead of sniffing extensions? 

The current approach requires sniffing URL extensions, which has at least two problems:
- We need to maintain these checks as we add more formats
- loaders.gl can also select format based on MIME type etc. in which case the extension may not be available.